crates/ruma-events/src/space/child.rs

```diff
@@ -44,6 +44,18 @@ pub struct ChildEventContent {
     pub suggested: Option<bool>,
 }
 
+impl ChildEventContent {
+    /// Creates a new `ChildEventContent` with the given list of candidate servers, ordering of
+    /// siblings and suggested flag.
+    pub fn new(
+        via: Option<Vec<ServerNameBox>>,
+        order: Option<String>,
+        suggested: Option<bool>,
+    ) -> Self {
```

Comment:
We often have an empty `fn new()` that just calls `Self::default()` (in that case it's not required to have `new` because `Default::default` also works, but it's good for discoverability).

I think we should do the same thing here, especially since there's a bool involved, which would be less readable as a regular argument rather than a field assignment.